.carve_ignore

```diff
@@ -9,3 +9,25 @@ athens.patterns/date
 athens.util/common-ancestor
 athens.events/pages
 athens.views.blocks.core/block-component
+athens.self-hosted.web.presence/next-id
```

Comment:
@neotyk left these ignored because I don't know if they are just not done yet or if they should be removed. Please advise.